packages/keystone/schema.js

```diff
@@ -167,11 +168,25 @@ async function getSchemaCtx (schemaObjOrName) {
     if (!SCHEMAS.has(name)) throw new Error(`Schema ${name} is not registered yet`)
     const schema = SCHEMAS.get(name)
     return {
+        type: schema._type,
         name: schema.name,
+        list: get(schema, ['_keystone', 'lists', schema.name], null),
         keystone: schema._keystone,
     }
 }
 
+/**
+ * Outputs gql schema
+ */
+function getSchema () {
```

Comment:
According to function logic, it returns map of contexts
```suggestion
function getSchemaContexts () {
```